packages/fuel-gauge/test/fixtures/forc-projects/token_abi/src/main.sw

```diff
@@ -3,9 +3,9 @@ library;
 use std::{address::Address, asset::*, contract_id::ContractId};
 
 pub struct TransferParams<Recipient> {
-    recipient: Recipient,
-    asset_id: AssetId,
-    amount: u64,
+    pub recipient: Recipient,
```

Comment:
Otherwise, we get an error:

```
fuel-gauge:pretest: ____
fuel-gauge:pretest:
fuel-gauge:pretest: error: Private struct field is inaccessible
fuel-gauge:pretest:   --> fuels-ts/packages/fuel-gauge/test/fixtures/forc-projects/token_contract/src/main.sw:37:22
fuel-gauge:pretest:    |
fuel-gauge:pretest: ...
fuel-gauge:pretest: 37 |                     .recipient,
fuel-gauge:pretest:    |                      ^^^^^^^^^ Private field "recipient" of the struct "TransferParams" is inaccessible in this module.
fuel-gauge:pretest:    |                      --------- help: Private fields can only be accessed within the module in which their struct is declared.
f
```

I don't have the related Sway PR for this, maybe [this one](https://github.com/FuelLabs/sway/pull/5613)? Not sure.